### DISCUSSION TITLE
Thumbnails on folders (with more than one book)

### DISCUSSION THREAD

**Discussion by thniels** [koreader/koreader#12245] [Q&A] [answered]
Is there a way to have Koreader show a folder thumbnail when in mosaic mode? It works brilliantly with individual books, but I would so love to be able to have a thumbnail on folders that contain book series etc. For instance, to be able to save an image file in a folder and have that be the designated thumbnail for that folder. If Koreader supports this, I must have missed the option somewhere. If not, I will hurry over to Issues and make a wish 😊.

**Comment by Frenzie** [comment, marked as answer]
Some discussion here: https://github.com/koreader/koreader/issues/12122
